src/coins.h

```diff
@@ -507,6 +507,17 @@ class CCoinsViewCache : public CCoinsViewBacked
      */
     CCoinsModifier ModifyCoins(const uint256 &txid);
 
+    /**
+     * Return a modifiable reference to a CCoins. Assumes that no entry with the given
+     * txid exists and creates a new one. This saves a database access in the case where
+     * the coins were to be wiped out by FromTx anyway.  This should not be called with
+     * the 2 historical coinbase duplicate pairs because the new coins are marked fresh, and
```

Comment:
There are no "2 historical coinbase duplicate pairs" in Zcash-derived chains, since we have enforced [BIP 30](https://github.com/bitcoin/bips/blob/master/bip-0030.mediawiki) ("height in coinbase") since launch. This was accidentally omitted for the genesis block, but that omission can have no effect on the BIP 30 goal of avoiding duplicate transactions. Delete this sentence from the comment.